example/README.md

```diff
@@ -0,0 +1,28 @@
+# CSI Mock Driver Example
```

Comment:
This may be better under the mock/ directory. We can point to it from the top level readme.

In the mock/readme.md, it would be also good to describe the high level limitations of the mock driver, such as:
* requires all the components to run on the same node
* only supports single node
* does not persist data across restarts

Reply:
Done